stablehlo/dialect/StablehloOps.td

```diff
@@ -1364,13 +1365,13 @@ def StableHLO_AllGatherOp : StableHLO_Op<"all_gather",
   }];
 
   let arguments = (ins
-    HLO_Tensor:$operand, /*all_gather_i1*/
+    Variadic<HLO_Tensor>:$operand, /*all_gather_i1*/
```

Comment:
`operands` and `results` here and below. We have some old ticket to unify naming (some ops are named `inputs` and/or `outputs`), but never got around to it, so let's make it explicit here to clarify.